pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1555,26 +1562,57 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Interfaces).To(HaveLen(2), "the VMI spec should feature 2 interfaces")
 			Expect(domain.Spec.Devices.Interfaces[1].Type).To(Equal("ethernet"), "Macvtap interfaces must be of type `ethernet`")
 		})
+		It("Should create network configuration for an interface using a binding plugin with tap domain attachment", func() {
+			bindingName := "BindingName"
+			v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+
+			iface1 := v1.Interface{Name: netName1, Binding: &v1.PluginBinding{Name: bindingName}}
+			net1 := v1.DefaultPodNetwork()
+			net1.Name = netName1
+
+			vmi.Spec.Networks = []v1.Network{*net1}
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{iface1}
+
+			domain := vmiToDomain(vmi, c)
+			Expect(domain).ToNot(BeNil())
+			Expect(domain.Spec.Devices.Interfaces).To(HaveLen(1))
+			Expect(domain.Spec.Devices.Interfaces[0].Type).To(Equal("ethernet"))
```

Comment:
Can you be specific and present the expected full config?

Reply:
All the other properties are not specific to the binding plugin with tap attachment and are covered (I hope) by other tests.

Reply:
Asserting the full spec makes sure there are no other side-effects.
But I will not insist.